lumicks/pylake/kymotracker/tests/test_kymotrackgroup_sources.py

```diff
@@ -8,7 +8,7 @@
 
 @pytest.fixture(scope="module")
 def kymos():
-    image = np.random.poisson(10, size=(10, 10, 3))
+    image = np.random.default_rng(1337).poisson(10, size=(10, 10, 3))
```

Comment:
Would turn `np.random.default_rng(number)` into an `rng` fixture itself, if possible since the pattern repeats so often.

Reply:
The scope of many of these fixtures is module (meaning they get constructed once for the whole module, not per test; which saves a lot of generation time). However, that means I can only use fixtures defined at the module scope inside of them (pytest limitation).

This in turn means that if I were to turn this generator into a fixture, all of the uses of that fixture would get the same generator (which only gets created once) which introduces a new source of randomness again (test order).

I could make it a free function, but it is already a free function, so I'm not sure that adds much. If it's merely about length, I could do a full import to remove the `np.random` part.

What do you think?

Reply:
Not worth it. Keep as is.